_sass/mei.scss

```diff
@@ -13,6 +13,10 @@ body {
 .content {
     flex: 1;
 }
+.container.content {
+    padding-left: 0.7rem;
+    padding-right: 0.7rem;
```

Comment:
May I ask, why you use relative values here instead of the 14px in the navbar? If we decide to change font-size it will break again.

Reply:
For no special reason, except that these `.container` padding values are declared as rem in spectre layout: https://github.com/music-encoding/music-encoding.github.io/blob/9a3234af316f94ea5131b2726c7c7155ca0ffa92/_sass/spectre/_layout.scss#L1-L8

with `$layout-spacing` pointing to 0.4rem
https://github.com/music-encoding/music-encoding.github.io/blob/9a3234af316f94ea5131b2726c7c7155ca0ffa92/_sass/spectre/_variables.scss#L52

But yes, maybe we should better use a derivative of `$layout-spacing` (=> `$layout-spacing-lg`) to have a relative relative value and not a fixed relative one? And use the same for the navbar instead of 14 px?

Reply:
Sounds plausible.

Reply:
what'S your take on this, go with what you already did or even add more improvement?

Reply:
Replaced values with spacing variables for header and cleaned up the scss selectors a bit.